### PR TITLE
Adds Falco-adapter documentation

### PR DESCRIPTION
**Pull request by anushkamittal20**
Adding a README.md for PolicyReport output being added in Falcosidekick to create/update reports for Falco events.

### CODE REVIEW

File: policy-report/falco-adapter/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Falco adapter
+The falco adapter runs as an output for falcosidekick when enabled, the adapter receives falco events (Falco ships with a default set of rules that check the kernel for unusual behavior) and produces N+1 reports (1 cluster-wide policy report and N namespace specific reports) based on the [Policy Report Custom Resource Definition](https://github.com/kubernetes-sigs/wg-policy-prototypes/tree/master/policy-report)
```
Comment (JimBugwadia) [inline policy-report/falco-adapter/README.md:2]: suggested updates:

The Falco Policy Report adapter receives Falco events and produces one or more [Policy Reports]([Policy Report Custom Resource Definition]). It runs as an optional output in (Falcosidekick)[https://github.com/falcosecurity/falcosidekick]. The adapter produces N+1 reports - 1 cluster-wide policy report and N namespace specific reports.

File: policy-report/falco-adapter/README.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+## Running
+
+**Prerequisites**: 
+* To run the Kubernetes cluster locally, tools like [kind](https://kind.sigs.k8s.io/) or [minikube](https://minikube.sigs.k8s.io/docs/start/) can be used. Here are the steps to run the falco adapater with a `kind` cluster.
```
Comment (JimBugwadia) [inline policy-report/falco-adapter/README.md:7]: typo in `adapater`

File: policy-report/falco-adapter/README.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+```sh
+helm install falco falcosecurity/falco --set falcosidekick.enabled=true --set falcosidekick.policyreport.enabled=true falcosidekick.policyreport.kubeconfig=~/.kube/config falcosidekick.policyreport.failthreshold=3 falcosidekick.policyreport.maxevents=10
+```
+Above can be configured according to specifications
+```sh
+kubectl port-forward svc/falco-falcosidekick 2801
+```
+5 can be configured according to specifications
+
+## Understanding the config options
```
Comment (JimBugwadia) [inline policy-report/falco-adapter/README.md:31]: Change to `Configuration`

File: policy-report/falco-adapter/README.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ * Once falcosidekick is enabled in policyreport output we have the following configurations available
+ ~~~
+1. Enabled = to enable policyreport output to create/update policyreports
+2. Kubeconfig = address to the file (default- ~/.kube/config)
+3. FailThreshold = events with priority above this threshold are mapped to Fail in PolicyReportSummary and "high" severity; rest are mapped to Warn in PolicyReportSummary (and severity "low" if event priority is below the threshold and "medium if it is equal to threshold" )
+4. MaxEvents = this specifies the maximum number of events in any of the N+1 reports; once the events go above this number the report start self pruning
+5. PruneByPriority = while pruning by default the events that came initially will be deleted (FIFO); by enabling this config the events that came initially of low priority are deleted before initial events of higher priority
+~~~
+
+## To get Report summary
```
Comment (JimBugwadia) [inline policy-report/falco-adapter/README.md:41]: Change to `Viewing reports`

File: policy-report/falco-adapter/README.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+```sh
+kubectl get clusterpolicyreports
+
+kubectl get policyreports --all-namespaces
+```
+
+## To get Reports
```
Comment (JimBugwadia) [inline policy-report/falco-adapter/README.md:48]: Consolidate with previous section

**Review by JimBugwadia — CHANGES_REQUESTED** [review]

File: policy-report/falco-adapter/README.md (hunk 1 above)
Comment (anushkamittal20) [reply policy-report/falco-adapter/README.md]: Hey @JimBugwadia , I have made these changes in the latest commit.
Thank you!

File: policy-report/falco-adapter/README.md (hunk 5 above)
Comment (anushkamittal20) [reply policy-report/falco-adapter/README.md]: Thank you very much for your suggestions @JimBugwadia, I have made all the above changes.

**Pull request merged**